fix compilation of intmath.f with gfortran 10.x
  Error: Arguments of iand have different kind type parameters at (1)
see https://github.com/wrf-model/WPS/pull/119
--- WPS/ungrib/src/ngl/g2/intmath.f.orig	2022-06-24 12:19:49.517587000 +0200
+++ WPS/ungrib/src/ngl/g2/intmath.f	2022-06-24 12:21:35.498978000 +0200
@@ -169,7 +169,7 @@
       ilog2_2=0
       i=i_in
       if(i<=0) return
-      if(iand(i,i-1)/=0) then
+      if(iand(i,i-1_2)/=0) then
          !write(0,*) 'iand i-1'
          ilog2_2=1
       endif
@@ -204,7 +204,7 @@
       ilog2_1=0
       i=i_in
       if(i<=0) return
-      if(iand(i,i-1)/=0) then
+      if(iand(i,i-1_1)/=0) then
          !write(0,*) 'iand i-1'
          ilog2_1=1
       endif
